### PR TITLE
Create change on group and feed creation

### PR DESCRIPTION
**Pull request by djabarovgeorge**
### What change does this PR introduce?

Steps to reproduce: 
Go to the workflow page
click on the Create Workflow button
choose password reset workflow (or any other) from pre-made workflows
update something if you want to update some content in that
Go to the changes page, find out changes related to this newly added workflow
Try to promote this change
You will get this error similar to the attached image

 

### Why was this change needed?

Its a bug.

### Other information (Screenshots)

The fix for this is to go back to workflow settings change the notification group to other and update and promote

### CODE REVIEW

**Review by LetItRock — APPROVED** [review]
can we please cover this with tests? thank you 🙏

File: apps/api/src/app/workflows/usecases/create-notification-template/create-notification-template.usecase.ts [hunk 1]
```diff
@@ -263,6 +278,18 @@ export class CreateNotificationTemplate {
...
       });
+
+      await this.createChange.execute(
+        CreateChangeCommand.create({
+          item: group,
+          environmentId: command.environmentId,
+          organizationId: command.organizationId,
+          userId: command.userId,
+          type: ChangeEntityTypeEnum.NOTIFICATION_GROUP,
+          parentChangeId: parentChangeId,
```
Comment (ainouzgali) [inline apps/api/src/app/workflows/usecases/create-notification-template/create-notification-template.usecase.ts:289]:
```suggestion
          parentChangeId,
```
@djabarovgeorge doesn't setting the parent here will make it into the same row? Like, it won't show as an independent change?

**Comment by djabarovgeorge** [comment]
> can we please cover this with tests? thank you 🙏

Unfortunately no.
In order to create a test that will simulate the issue we need to make a request to the `dev` environment, which should be fine because we took it into account in blueprint feature discovery.
However, it is not possible to create such a test before we fix [issue](https://linear.app/novu/issue/NV-3188/incorrect-creation-of-the-blueprint-on-cross-origin) i found while fixing this issue

File: apps/api/src/app/workflows/usecases/create-notification-template/create-notification-template.usecase.ts (hunk 1 above)
Comment (djabarovgeorge) [reply apps/api/src/app/workflows/usecases/create-notification-template/create-notification-template.usecase.ts:289]: Yes, and it is the desired behavior i think we should merge all the possible changes into one workflow change in order to reduce potential issues like we had in the past
Do you have another opinion on the matter?

Comment (ainouzgali) [reply apps/api/src/app/workflows/usecases/create-notification-template/create-notification-template.usecase.ts:289]: Its just not the current behavior we have for notification group. They are showing separately ->notification group change can be promoted as itself. But if you promote the notification template change  it promotes both changes

**Pull request merged**